LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIPersonnel_SoldierListItem_LW.uc

```diff
@@ -28,10 +26,7 @@ simulated function UIButton SetDisabled(bool disabled, optional string TooltipTe
 {
 	super.SetDisabled(disabled, TooltipText);
 	UpdateDisabled();
-	// KDM : Originally, UpdateItemsForFocus was called with the parameter 'false'; however, this was problematic
-	// since ability text colour would no longer depend upon the list item's focus state; it would always 'assume'
-	// it was unfocused. In order to take the list item's focus into account, we need to send in bIsFocused.
-	UpdateItemsForFocus(bIsFocused);
+	UpdateItemsForFocus();
```

Comment:
In the commit message you say it's important to call `OnReceiveFocus()` or `OnLoseFocus()` before `UpdateItemsForFocus()`, but that doesn't happen here. Some clarification would be helpful.

Reply:
I've updated the commit message and message here.

`Within OnReceiveFocus and OnLoseFocus, it is important to call their super variant, super.OnReceiveFocus or super.OnLoseFocus, before UpdateItemsForFocus. This is because UpdateItemsForFocus makes use of a UIPanel's focus status, defined by bIsFocused and updated via UIPanel.OnReceiveFocus and UIPanel.OnLoseFocus. Originally, within LW2, UpdateItemsForFocus was passed a true or false value to represent focus state; hence, function call order was unimportant.`

`SetDisabled` is not concerned with "setting" focus status; therefore, it need not deal with `OnReceiveFocus` nor `OnLoseFocus`. It will take whatever the list item's focus state is, represented through `bIsFocused`, and update accordingly.